src/index.js

```diff
@@ -10,8 +10,8 @@ export default function (source, sourceMap) {
     srcMap = inlineSourceMap.sourcemap;
   }
 
-  const userOptions = loaderUtils.parseQuery(this.query);
-  const instrumenter = istanbulLibInstrument.createInstrumenter(
+  const userOptions = loaderUtils.getOptions(this);
```

Comment:
This is safe as passing null as an arg to Object.assign just has it skip it without an error `Object.assign({foo: 'bar'}, null) == {foo: 'bar'}`

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign